x-pack/docs/en/rest-api/ml/validate-job.asciidoc

```diff
@@ -28,7 +28,6 @@ see <<ml-job-resource,Job Resources>>.
 You must have `manage_ml`, or `manage` cluster privileges to use this API.
 For more information, see
 {xpack-ref}/security-privileges.html[Security Privileges].
-//<<privileges-list-cluster>>.
```

Comment:
This same line is in validate-detector.asciidoc I guess that can also be removed